frontend/src/pages/notebookController/screens/server/StartServerModal.tsx

```diff
@@ -51,9 +53,8 @@ const StartServerModal: React.FC<StartServerModalProps> = ({ open, spawnInProgre
     }
   }, [open]);
 
-  const notebookLink = notebook?.metadata.annotations?.['opendatahub.io/link'];
-
-  const spawnFailed = spawnStatus?.status === AlertVariant.danger;
+  const spawnFailed =
+    spawnStatus?.status === AlertVariant.danger || spawnStatus?.status === AlertVariant.warning;
```

Comment:
Could you tell me why do you add `warning` back to the spawn failed flag? The warning currently only happens when
```
case 'Killing': {
  currentEvent = 'Stopping container oauth-proxy';
  status = EventStatus.WARNING;
  break;
}
```
We don't want to change the `Cancel` button to `Close` which makes the user think the spawning failed in this situation.

Reply:
Oops, bad conflict resolution. My bad. I read the diff wrong. Let me update.